test/mining.js

```diff
@@ -111,9 +111,10 @@ describe("Mining", function() {
         jsonrpc: "2.0",
         method: "evm_mine",
         id: new Date().getTime()
-      }, function(err) {
+      }, function(err, result) {
         if (err) return reject(err);
-        accept();
+        if(result.result !== '0x0') return reject(new Error("evm_mine should return result = '0x0':" + JSON.stringify(result)))
```

Comment:
This works fine, but it's more idiomatic to write `assert.notDeepEqual(result.result, '0.0')`

Reply:
No need to resubmit - I'll change it when I merge